### ISSUE TITLE
[Bug]: Pydantic BaseSettings have moved to pydantic-settings

### ISSUE THREAD

**Issue by bryanmoy** [pathwaycom/cookiecutter-pathway#1]
### Steps to reproduce

1. Start a new cookiecutter project: `cookiecutter https://github.com/pathwaycom/cookiecutter-pathway`.
2.`cd` into the project directory.
3. Build the Docker container: `docker compose -f local.yml build`.
4. Start the Docker container: `docker compose -f local.yml up`.

### Relevant log output

```text
new_pathway_app-pathway_app-1  | Traceback (most recent call last):
new_pathway_app-pathway_app-1  |   File "/new_pathway_app/run.py", line 1, in <module>
new_pathway_app-pathway_app-1  |     from new_pathway_app.config import get_settings
new_pathway_app-pathway_app-1  |   File "/new_pathway_app/new_pathway_app/config.py", line 5, in <module>
new_pathway_app-pathway_app-1  |     from pydantic import BaseSettings
new_pathway_app-pathway_app-1  |   File "/usr/local/lib/python3.11/site-packages/pydantic/__init__.py", line 218, in __getattr__
new_pathway_app-pathway_app-1  |     return _getattr_migration(attr_name)
new_pathway_app-pathway_app-1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
new_pathway_app-pathway_app-1  |   File "/usr/local/lib/python3.11/site-packages/pydantic/_migration.py", line 294, in wrapper
new_pathway_app-pathway_app-1  |     raise PydanticImportError(
new_pathway_app-pathway_app-1  | pydantic.errors.PydanticImportError: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.4/migration/#basesettings-has-moved-to-pydantic-settings for more details.
new_pathway_app-pathway_app-1  |
new_pathway_app-pathway_app-1  | For further information visit https://errors.pydantic.dev/2.4/u/import-error
new_pathway_app-pathway_app-1 exited with code 1
```

### What did you expect to happen?

No errors.

### Version

8394e7dd9deef2d23c8ae532b1a999c8a001ccac

### OS

MacOS

### On which CPU architecture did you run Pathway?

None

**Comment by bryanmoy** [comment]
Hello, 

This issue was quite straightforward so I went ahead and forked the project [here](https://github.com/bryanmoy/cookiecutter-pathway) and fixed the issue in [this commit](https://github.com/pathwaycom/cookiecutter-pathway/compare/main...bryanmoy:cookiecutter-pathway:main).

**Comment by lewymati** [comment]
Resolved by #2
